tests/RobotFramework/tests/cumulocity/configuration/configuration_operation_multi_cloud.robot

```diff
@@ -5,26 +5,21 @@ Library             ThinEdgeIO
 Library             Cumulocity
 
 Suite Setup         Suite Setup
-Suite Teardown      Get Logs    name=${PARENT_SN}
-Test Setup          Test Setup
+Suite Teardown      Get Logs    name=${DEVICE_SN}
 
-Test Tags           theme:configuration    theme:childdevices
+Test Tags           theme:configuration
 
 
 *** Variables ***
-${PARENT_SN}            ${EMPTY}
+${DEVICE_SN}            ${EMPTY}
 ${SECOND_DEVICE_SN}     ${EMPTY}
 
 
-*** Test Cases ***    DEVICE    EXTERNALID    CONFIG_TYPE    DEVICE_FILE    FILE    PERMISSION    OWNERSHIP
-#
-# Get configuration
-#
-
+*** Test Cases ***
```

Comment:
```suggestion
*** Test Cases ***
# Since the test framework does not have access to multiple cloud tenants,
# multi cloud profile support is tested by connecting the same device to the same cloud
# using different external identities configured using different profiles.
# This setup results in a single "device" having two device twin identities in the cloud.
# That same "device" can be controlled via either twins.
```
Adding some commentary would be nice to help others looking at this test and going: "where are those multiple cloud connections"?